recipes/reflect-cpp/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, export_conandata_patches, apply_conandata_patches
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.51.1"
+
+class ReflectCppConan(ConanFile):
+    name = "reflect-cpp"
+    description = "C++-20 library for fast serialization, deserialization and validation using reflection"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getml/reflect-cpp"
+    topics = ("reflection", "serialization", "memory", "json", "xml", "flatbuffers", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_json" : [True, False],
+        "with_xml" : [True, False],
+        "with_flatbuffers" : [True, False],
+        "with_yaml": [True, False],
+    }
+    default_options = {
+        "with_json" : False,
+        "with_xml" : False,
+        "with_flatbuffers" : False,
+        "with_yaml" : False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "17",
+            "msvc": "193",
+            "gcc": "11.4",
+            "clang": "16",
+            "apple-clang": "15",
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.get_safe("with_json", True):
```

Comment:
They can stay like this for now and maybe be changed in the future to avoid recompilations now, but options can not be missing with how the recipe is currently setup, so just doing `self.options.with_json` would work :)

Reply:
Might as well change them now!